README.adoc

```diff
@@ -462,18 +457,18 @@ Let's work together to make this a welcoming, professional, inclusive, and safe
 Copyright (C) 2012-2021 Dan Allen, Sarah White, Ryan Waldron, and the individual contributors to Asciidoctor.
 Use of this software is granted under the terms of the MIT License.
 
-See the {uri-license}[LICENSE] for the full license text.
+See the {url-license}[LICENSE] for the full license text.
 
 == Authors
 
-*Asciidoctor* is led by https://github.com/mojavelinux[Dan Allen] and https://github.com/graphitefriction[Sarah White] and has received contributions from {uri-contributors}[many individuals] in Asciidoctor's awesome community.
-The project was initiated in 2012 by https://github.com/erebor[Ryan Waldron] and based on {uri-prototype}[a prototype] written by https://github.com/nickh[Nick Hengeveld].
+*Asciidoctor* is led by https://github.com/mojavelinux[Dan Allen] and https://github.com/graphitefriction[Sarah White] and has received contributions from {url-contributors}[many individuals] in Asciidoctor's awesome community.
+The project was initiated in 2012 by https://github.com/erebor[Ryan Waldron] and based on a prototype written by https://github.com/nickh[Nick Hengeveld].
 
 *AsciiDoc* was started and maintained by Stuart Rackham from https://github.com/asciidoc/asciidoc/blob/master/CHANGELOG.txt[2002 to 2013] and has received contributions from many individuals in the https://github.com/asciidoc/asciidoc/graphs/contributors[AsciiDoc community].
 
 == Trademarks
 
-AsciiDoc is a Trademark of the Eclipse Foundation, Inc.
+AsciiDoc is a trademark of the Eclipse Foundation.
```

Comment:
Is dropping the ", Inc" part of the transition of the Eclipse Foundation to Europe? If so, we need to make the same change to the docs UI.

Reply:
I got clarification from legal. Initially, I made this change because in the Eclipse handbook and many of the other projects' notices, they format/phrase the attribution without the "Inc". But you were correct, it should be "Eclipse Foundation, Inc." Also, AsciiDoc is a registered trademark, so attribution requires the (R) as well as the first/most prominent usage of it, which I added in the first paragraph.  AsciiDoc Language (when writing about as the specification project, which we don't do on this page but I added attribution anyway) gets the (TM).

Reply:
BTW, you can find the full list of trademarks here: https://www.eclipse.org/legal/trademarks.php#attribution and the usage guidelines for them here: https://www.eclipse.org/legal/logo_guidelines.php